samples/philosophers/src/main.c

```diff
@@ -83,7 +83,7 @@
 /* end - control behaviour of the demo */
 /***************************************/
 
-#define STACK_SIZE 768
+#define STACK_SIZE 1024
```

Comment:
I'm trying to understand something here.
IIRC, On ARM, the guard region for stacks is taken from the existing stack size. It is not added like on x86.
Wouldn't a stack size of 1024 be entirely consumed by the guard area, leaving no room for the actual stack buffer?

Reply:
yes. if the MPU guard is enabled it will consume 1K out of the stack size.
but I want to create a warning that will inform the users of this limitation when we refactor qemu_cortex_m3.

Reply:
MPU Stack guard is not enabled by default. Hence this works.

Reply:
MPU stack guard is now enabled by default for tests.
We don't have the 1K limitation any more, correct?